### ISSUE TITLE
Recurrent event by months or years

### ISSUE THREAD

**Issue by catsoft** [davish/obsidian-full-calendar#546]
### Submit on Discussions

- [X] I will submit my feature request as a [Discussion](https://github.com/davish/obsidian-full-calendar/discussions/new?category=feature-requests).

**Comment by Furupuru** [comment]
Also looking for this functionality, yearly in particular

**Comment by remi-scarlet-1** [comment]
Yeah, came here looking for this too.
Might even get into the code myself and do a pull request, if I find the time...

**Comment by alvaromartin** [comment]
+1 for recurring yearly

**Comment by ReaderGuy42** [comment]
Any news on this?

**Comment by andrewfirsenko** [comment]
+1

**Comment by GitOnTop** [comment]
Yes, please! Having yearly recurring events would make Full Calendar "complete", IMO.

**Comment by garciaErick** [comment]
Hmm I am putting a +1, if it is using the Full calendar js under the hood, cant we just expose the RRule settings?
https://fullcalendar.io/docs/rrule-plugin

I tried setting this in my event but it didnt work, it does not render my recurring yearly event

**Comment by Xosrov** [comment]
I used some AI bullshit to add in a patch that adds [yearly](https://github.com/Xosrov/obsidian-full-calendar/commit/cbba14b9cb3a1b0af4b23375c896af99e76d1d5d) in the calendar view. In case anybody cares

**Comment by YouFoundJK** [comment]
This should now be possible here [latest version](https://github.com/YouFoundJK/plugin-full-calendar/releases/latest), if you are still looking for a fix.

Update: Currently not in the release - had to take it down for some guidelline compatibility reasons! Sorry.
Edit: Now its working again ;p
